src/index.js

```diff
@@ -1,51 +1,135 @@
 
 import './style.css';
 const divContainer = document.querySelector(".books-container");
+const inputDiv = document.querySelector("#title");
+const inputSubmit = document.querySelector("#submit");
 
 
+let todoList = [];
+let isEdit = false
+let editId = null;
 
-const todoList = [
-    {
-      description: 'Go shopping for lunch',
-      completed: false,
-      index: 0,
-    },
-    {
-      description: 'Take kids to school',
-      completed: false,
-      index: 1,
-    },
-    {
-      description: 'Work on coding challenges',
-      completed: false,
-      index: 2,
-    },
-    {
-        description: 'Go to the gym',
-        completed: false,
-        index: 3,
-      },
-    {
-      description: 'Attend workshop',
-      completed: false,
-      index: 4,
-    },
-  ];
-
-  const displayTodo = ({ description, index }) => {
-    const divElement = document.createElement('div');
-    divElement.className = 'first-item';
-    divElement.innerHTML = `
+console.log(editId)
+
+
+
+  const setLocalStorage = (todoList)=>{
+    localStorage.setItem('formInputs', JSON.stringify(todoList));
+  }
+
+  const getLocalStorage = ()=>{
+    if (localStorage.getItem('formInputs') !== null) {
+      todoList = JSON.parse(localStorage.getItem('formInputs'));
+      } else {
+      todoList = [];
+
+      }
+      return todoList;
+  }
+
+  const displayTodo = ({ description, index}) => {
+    
+      const divElement = document.createElement('div');
+
+      divElement.className = 'first-item';
+      divElement.innerHTML = `
       <div class="item-details">
       <input type="checkbox" id="" name="" value=""> <h4 class="item-desription">${description}</h4>
       </div>
-      <i class="fa-solid fa-trash-can" id"${index}"></i>
-      `;
+      
+      <button class="edit">edit</button>
```

Comment:
Kindly note that your edit button does not work. It just adds a new element instead of editing the one clicked. See the video attached.

https://user-images.githubusercontent.com/48546992/199699632-a304fd1b-8054-4072-b5b5-886c3c8ff821.mov

Reply:
Fixed.